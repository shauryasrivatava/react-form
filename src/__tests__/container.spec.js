import renderer from 'react-test-renderer';
import Container from '../components/Container';
import React from 'react';
import { mount } from 'enzyme';
import enzymeConfig from '../../enzymeConfig'

describe('<Container />', function(){

    it('should render correctly', function(){
        const component = renderer.create(<Container/>);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });

    it('should have <div> with container class', function(){
        const component = mount(<Container />);
        expect(component.find('div').hasClass('container')).toBe(true);
    });
})