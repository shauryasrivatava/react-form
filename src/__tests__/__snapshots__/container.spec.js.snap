// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> should render correctly 1`] = `
<div
  className="container"
/>
`;
